sklearn/svm/tests/test_svm.py

```diff
@@ -1100,11 +1100,15 @@ def test_linear_svm_convergence_warnings():
     warning_msg = "Liblinear failed to converge, increase the number of iterations."
     with pytest.warns(ConvergenceWarning, match=warning_msg):
         lsvc.fit(X, Y)
+    # Check that we have an n_iter_ attribute with int type as opposed to a
+    # numpy array or an np.int32 so as to match the docstring.
+    assert isinstance(lsvc.n_iter_, int)
     assert lsvc.n_iter_ == 2
```

Comment:
This test would pass for the wrong reason when `n_iter_ == np.array([2, 2, 2], np.int32)`...